cgi/generate_sample_import_file.pl

```diff
@@ -169,18 +191,8 @@
 			}
 		}
 
-		my $example = lang($field_id . "_example");
-
-		if ($example ne "") {
-
-			my $example_title = lang("example");
-
-			# Several examples?
-			if ($example =~ /,/) {
-				$example_title = lang("examples");
-			}
-			$comment .= $example_title . " " . $example . "\n\n";
-		}
+		my $line_lc = lc($request_ref->{lc} || 'en');
+    my $example_value = $example_values_by_language_and_header{$line_lc}{$field_ref->{text}} // '';
```

Comment:
The issue is that $field_ref->{text} can be in another language than English, but in the sample configuration file we have English.
The best solution is to add a row with field ids in the example configuration file. I made a PR based on yours here: https://github.com/openfoodfacts/openfoodfacts-server/pull/9985